odml/dtypes.py

```diff
@@ -200,16 +200,15 @@ def datetime_set(value):
 def boolean_get(string):
     if string is None:
         return None
-    if type(string) is bool:
-        string = str(string)
-    string = string.lower()
-    truth = ["true", "t", "1"]  # be kind, spec only accepts True / False
+    if type(string) is unicode:
+        string = string.lower()
+    truth = ["true", "1", True]  # be kind, spec only accepts True / False
```

Comment:
@achilleas-k  Will add `"t"` and `"f"` back :+1: .. What I had interpreted from the conversation in issue #96 was that **everything** that is not zeros should be treated as `True` (But still, thought of keeping `"true"` and `"false"`, as they invariably relate to a `bool` type, but `"t"` and `"f"`, not always).